### DISCUSSION TITLE
Is there audio support ?

### DISCUSSION THREAD

**Discussion by SeleDreams** [friction2d/friction#81] [Q&A]
Hi, I've been trying out the software, my main use case would be to make motion graphic animations synced to music for music videos.
Because of it I've been wondering if the software had audio support ? since it is kind of essential for my use case.

**Comment by IcedQuinn** [comment]
It would be nice to have something basic. OpenToonz allows you to put sound clips in, as does Blender, and I think even Nuke has very basic support for importing a soundtrack for this task.

**Reply by rodlie** [reply]
Friction has audio support, it supports audio included with video clips and standalone audio clips.
